### ISSUE TITLE
[blink-dev] Intent to Remove: Support for embedded credentials in subresource requests.

### ISSUE THREAD

**Issue by ghost** [getsentry/sentry#4808]
Unsure if this will affect Sentry but worth looking into before it's yanked. (This seems relevant for raven-js et al.?)

https://groups.google.com/a/chromium.org/d/msg/blink-dev/lx-U_JR2BF0/Hsg1fiZiBAAJ

**Comment by mattrobenolt** [comment]
If this happens, this will be annoying for us. To us, this isn't really a username/password thing, it's just how we format our API keys. :( We have a similar issue with the Google Play store already triggering a warning for it. See https://forum.sentry.io/t/google-play-warning-your-app-contains-url-credentials-which-is-often-unintentional/766 for context.

**Comment by leonelwilliams** [comment]
It may be worth adding your voice as a Sentry member to the thread (I'm not affiliated with the Chromium/blink project - just happened upon this email)

**Comment by leonelwilliams** [comment]
Looks like this may well happen: https://groups.google.com/a/chromium.org/d/msg/blink-dev/lx-U_JR2BF0/8wund4yFCgAJ

**Comment by mattrobenolt** [comment]
@leonelwilliams So reading into this more, this shouldn't actually affect us. They seem to be intending to block making subresource requests that are using credentials as a part of the request.

On our case, we're not doing this. raven-js just parses the DSN and uses them to make a normal request and passes the credentials as a querystring variable, in our case.

So from my understanding, this should have 0 impact on us since we don't actually use the "credentials" bits to make requests in a traditional manner.

/cc @mikewest if we can chime in here, would be good to get his blessing. <3

tl;dr, our "DSN"s look like `https://ABC@sentry.io/1`. So a user configures Sentry with `Raven.config('https://ABC@sentry.io/1)` in their JavaScript. But within our library, we use that information to make a request to `https://sentry.io/api/1/store/?sentry_key=ABC` without using Basic auth.

**Comment by mikewest** [comment]
Hi! Skimming through the code on `sentry.io`, I see a script block like the following:

```
Raven.config('https://ABCDEFG@sentry.io/11276', {
    ...
}).install();
```

If that's the only place you're using this URL format, and you convert them to a URL that doesn't directly include a username/password when making requests, then you should be fine!

I intend to turn it on by default in Chrome ~59. If it would be helpful, I can probably land the behavior behind a flag for y'all to play with in Canary next week?

**Comment by mattrobenolt** [comment]
Thanks for confirming @mikewest. From my understanding, we would have been good too. Just wanted to make sure.

I'll follow along with this and make sure to double check when it's available in Canary. ❤️

**Comment by BYK** [comment]
Closing this issue due to staleness. Feel free to comment here if you think we should still work on this.
